mongo/options/autoencryptionoptions.go

```diff
@@ -144,13 +144,16 @@ func (a *AutoEncryptionOptions) SetTLSConfig(tlsOpts map[string]*tls.Config) *Au
 }
 
 // SetEncryptedFieldsMap specifies a map from namespace to local EncryptedFieldsMap document.
+// EncryptedFieldsMap is used for Queryable Encryption.
+// Queryable Encryption is in Public Technical Preview.
```

Comment:
What should "Public Technical Preview" mean to users reading the documentation? For example, does it mean they should not rely on these APIs for production use yet, or that the API may change in a future version of the driver, or something else?

Reply:
Good question. I interpret "Public Technical Preview" to mean users should not rely on Queryable Encryption for production use, and that the API is subject to breaking changes. Posted in DRIVERS-2349 to confirm. Updated comments.